r/R/read_parquet.R

```diff
@@ -0,0 +1,10 @@
+#' Read parquet file from disk
```

Comment:
This needs the license headers, you can just copy from any other R file. 
https://travis-ci.org/apache/arrow/jobs/470349974#L493